### ISSUE TITLE
Operon "copy" seem to duplicate some tags

### ISSUE THREAD

**Issue by zuzzurro** [quodlibet/quodlibet#3344]
Steps to reproduce
------------------

Using operon to copy tags from a source FLAC audio file to a destination MP4 

Expected Output
---------------

The set of tags managed by operon should appear in the destination

Actual Output
-------------

Some tags got duplicated

Test System
-----------
use picard to tag the source flac
use qaac to transcode the flac to m4a
then:
operon clear -a dst.m4a
operon copy --ignore-errors source.flac dst.m4a

use picard or mp3tags to check the output. Some tags are duplicated

![image](https://user-images.githubusercontent.com/6942457/79239650-1b247600-7e71-11ea-8938-4092963757bb.png)

I'm still not clear what's exactly going on. But it used to work until some time ago, maybe it changed when I moved to Fedora 32 beta, I will try to figure out when it started breaking.

### Which version of Quod Libet?

4.3

### Which operating system 

Fedora 32 beta

### If it's audio-related, what back-end?

Additional Information
----------------------

**Comment by lazka** [comment]
Do they also show up with "operon list" ?

**Comment by zuzzurro** [comment]
I dug a bit further on this and maybe there's a hint. By opening a file tagged with operon with foobar2000 here's what I see:

![image](https://user-images.githubusercontent.com/6942457/79250929-c3413b80-7e7f-11ea-9723-02576b118751.png)

It seems there are two versions of some musicbrains tags. One with underscores and one with blanks. Do you know what's going on? Some tools seem to be confused by this.

**Comment by zuzzurro** [comment]
Here's the Foobar view of the same tags I showed earlier from mp3tags:

![image](https://user-images.githubusercontent.com/6942457/79251501-84f84c00-7e80-11ea-8c36-a48a9768b1ff.png)

What's really wrong is that if I feed these files back to picard it puts everything under "[non-album tracks]" with this error:

D: 19:00:45,045 /usr/lib64/python3.8/site-packages/picard/tagger._file_loaded:456: <MP4File "01. Read My Mind (Pet Shop Boys 'Stars Are Blazing' Mix).m4a"> has only recording MBID (ab5b95f0-128c-471f-a5cf-bed6a76ff49b), moving to non-album track...

**Comment by lazka** [comment]
The problem could be that we don't read all m4a tags, and so "clear" only deletes the ones QL knows about.

**Comment by zuzzurro** [comment]
Yes, I think that's it. Qaac copies by default the flac tags, operon clear doesn not clear stuff it doesn't know and finally operon copy uses a different standard naming.
Transcoding and tagging a stupid audio file really is more difficult than it should be. The workaround I'm thinking is to go back to encode to an intermediate wav file (not tags) and then qaac it. This leaves open the issue of copying the cover image though, that operon doesn't manage well (see #1935)

**Comment by zuzzurro** [comment]
I had created three issues in 2016, none of them have made any progress although you seemed not to be against them. Would you still consider implementing them?
